src/commands/helm-conftest.yaml

```diff
@@ -0,0 +1,18 @@
+---
+parameters:
+  chart:
+    type: "string"
+  policy_path:
+    description: Path to the Rego policy files directory.
+    type: string
+    default: policy
+description: Test conftest policies
+steps:
+  - run: |
+      mkdir -p policy
+      conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.16-deprek8ion.rego
+      conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.17-deprek8ion.rego
+      conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.18-deprek8ion.rego
+      conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.19-deprek8ion.rego
+      conftest pull --policy policy https://raw.githubusercontent.com/swade1987/deprek8ion/6b1c53db2d48c9361c61d73dab3594f01324f68a/policies/1.20-deprek8ion.rego
```

Comment:
In case `policy_path` is different than default there is no need to create the policy directory and pull all the policy files

Reply:
Fixed.